abzal.tankibayev/HomeWork05/wwwroot/server.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+
+const express = require('express');
+const Task = require('./controllers/tasks');
+const app = express();
+const PORT = 8888;
```

Comment:
Рекомендую добавить небольшую возможность кастомизации, например так:
```suggestion
const PORT = process.env.PORT || 8888;
```